test/testlib/constants.go

```diff
@@ -24,6 +24,7 @@ const TEARDOWN_DATABASE = "database"
 const TEARDOWN_RESTORE = "database"
 const TEARDOWN_SECRETS = "secrets"
 const TEARDOWN_YCSB = "ycsb"
+const TEARDOWN_INSIGHTS = "insights"
```

Comment:
I will make this comment here, but it applies to a few more places. Can you please purge the term "insights" from places that are actually the collector? That is: variables, constants, function names, etc.

I wonder, is there anything in this review that relates to "Insights" the monitoring stack? Or are all changes here related to the collector? I think that you check log files? You don't output anything to Influx?

If this review has nothing to do with a particular output target, I suggest that we remove `all` mentions of Insights.

Reply:
"Insights" is used here as we are deploying a dynamic output plugin similar to the one that Insights will deploy. Here we use "outputs.file" for simplicity to avoid installing InfluxDB. One of the things tested here is that a dynamic output plugin similar to the one deployed by Insights chart will be loaded.

I've removed "insights" in all the places where I think it's not relevant.